lib/omise/res/OmiseApiResource.php

```diff
@@ -135,11 +131,14 @@ protected function execute($url, $requestMethod, $key, $params = null)
     {
         // If this class is execute by phpunit > get test mode.
         if (preg_match('/phpunit/', $_SERVER['SCRIPT_NAME'])) {
-            $result = $this->_executeTest($url, $requestMethod, $key, $params);
+            $client = new \Omise\Client\UnitTestClient;
```

Comment:
Note, this is not an ideal way.
But to limit the area of the relevant code that this pull request is changing, I'm leaving it as it is by intention to focus on the organizing code first.